aptos-move/framework/aptos-framework/sources/code.move

```diff
@@ -214,10 +214,18 @@ module aptos_framework::code {
         );
 
         let registry = borrow_global_mut<PackageRegistry>(code_object_addr);
-        vector::for_each_mut<PackageMetadata>(&mut registry.packages, |pack| {
+        vector::for_each_mut(&mut registry.packages, |pack| {
             let package: &mut PackageMetadata = pack;
             package.upgrade_policy = upgrade_policy_immutable();
         });
+
+        // We unfortunately have to make a copy of each package to avoid borrow checker issues as check_dependencies
+        // needs to borrow PackageRegistry from the dependency packages.
+        // This would increase the amount of gas used, but this is a rare operation and it's rare to have many packages
+        // in a single code object.
+        vector::for_each(registry.packages, |pack| {
+            check_dependencies(code_object_addr, &pack);
```

Comment:
This `check_dependencies ` function returns `allowed_module_deps` that are normally used in the native bytecode dep verification (https://github.com/aptos-labs/aptos-core/blob/3b50ea6e4c3170cfaaa02193b0334cbabeaae875/aptos-move/framework/aptos-framework/sources/code.move#L266-L266). Is it fine to ignore the return value?

Reply:
Yeah here in this freeze function we only care about visibility checks as other checks should have already been performed when this package was first deployed or last updated